test/tests/itemsTest.js

```diff
@@ -2,6 +2,7 @@ describe("Zotero.Items", function () {
 	var win, collectionsView, zp;
 	
 	before(function* () {
+		this.timeout(10000);
 		win = yield loadZoteroPane();
```

Comment:
This was taking too long when running items tests directly without running other tests first.